<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmProgressButton</class>
 <widget class="QWidget" name="frmProgressButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget1" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QEXTProgressButton" name="progressButton1" native="true"/>
    </item>
    <item>
     <widget class="QEXTProgressButton" name="progressButton2" native="true"/>
    </item>
    <item>
     <widget class="QEXTProgressButton" name="progressButton3" native="true"/>
    </item>
    <item>
     <widget class="QEXTProgressButton" name="progressButton4" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>481</width>
     <height>111</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QEXTProgressButton" name="progressButton5" native="true"/>
    </item>
    <item>
     <widget class="QEXTProgressButton" name="progressButton6" native="true"/>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QEXTProgressButton</class>
   <extends>QWidget</extends>
   <header>qextProgressButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
